framework/behaviors/SluggableBehavior.php

```diff
@@ -157,7 +159,7 @@ protected function getValue($event)
      * Checks whether the new slug generation is needed
      * This method is called by [[getValue]] to check whether the new slug generation is needed.
      * You may override it to customize checking.
-     * @return bool
+     * @return boolean
```

Comment:
This is wrong change. Should be `bool` because `boolean` can't be used as return type in PHP 7+.